.github/workflows/checks.yml

```diff
@@ -248,18 +248,19 @@ jobs:
       - name: Harden runner
         uses: step-security/harden-runner@18bf8ad2ca49c14cbb28b91346d626ccfb00c518 # v2.1.0
         with:
-          disable-sudo: true
-          egress-policy: block
-          allowed-endpoints: >
-            api.github.com:443
-            artifactcache.actions.githubusercontent.com:443
-            codecov.io:443
-            github.com:443
-            nodejs.org:443
-            objects.githubusercontent.com:443
-            registry.npmjs.org:443
-            storage.googleapis.com:443
-            uploader.codecov.io:443
+          disable-sudo: false
+          egress-policy: audit
+          # egress-policy: block
+          # allowed-endpoints: >
+          #   api.github.com:443
+          #   artifactcache.actions.githubusercontent.com:443
+          #   codecov.io:443
+          #   github.com:443
+          #   nodejs.org:443
+          #   objects.githubusercontent.com:443
+          #   registry.npmjs.org:443
+          #   storage.googleapis.com:443
+          #   uploader.codecov.io:443
```

Comment:
This change should be reverted with an updated list of `allowed-endpoints` before this gets merged.

Reply:
Resolved by a68c5917467c75688f5b985b569b6b9702579ad0